Gemfile

```diff
@@ -1,12 +1,12 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.7.0'
+ruby '2.6.5'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.4'
 # Use postgresql as the database for Active Record
-gem 'pg', '>= 0.18', '< 2.0'
+gem 'sqlite3'
```

Comment:
- [ ] **IMPORTANT** Since you are going to push your project to Heroku in the end, I highly suggest reverting back to using `pg` instead of `sqlite3` as Heroku requires you to use `pg` not `sqlite3`. So reverting to `pg` now will save you a lot of overhead when pushing to Heroku.